### ISSUE TITLE
MapView InitialRegion ignored when using google provider on iOS

### ISSUE THREAD

**Issue by AbdullahWali** [expo/expo#1643]
### Environment
Environment:
  OS: Windows 10
  Node: 6.11.2
  Yarn: 1.5.1
  npm: 3.10.10
  Watchman: Not Found
  Xcode: N/A
  Android Studio: Not Found

Packages: (wanted => installed)
  expo: ^26.0.0 => 26.0.0
  react: 16.3.0-alpha.1 => 16.3.0-alpha.1
  react-native: https://github.com/expo/react-native/archive/sdk-26.0.0.tar.gz => 0.54.2
  Targetting iOS

### Steps to Reproduce

I’m trying to use the MapView with Google Maps on iOS. However the initialRegion that I set is being ignored and the map always loads zoomed out to the maximum level when provider is set to “google” (Works normally with iOS maps).

I found this issue on the react-native-maps repo which addresses the exact same problem: https://github.com/react-community/react-native-maps/issues/2140

It is supposedly fixed in sdk26, but I have sdk26 and I’m still getting the same problem.

### Reproducible Demo
reproducible demo is from the previous link to the issue in the react-native-maps repo. The link also includes Expected Behaviour and Actual Behaviour 
https://snack.expo.io/@evoluzione/map-zoom

**Comment by ccheever** [comment]
Also see forum thread here:
https://forums.expo.io/t/mapview-initialregion-ignored-when-using-google-provider-on-ios/8805

This doesn’t work in SDK 26 (verified that here) but it sounds like the fix may have been made in master of React Native Maps so it should get propagated to SDK 27 I think.

Repro of it not working in SDK 26. https://snack.expo.io/SyOHzQpiG 2

**Comment by AbdullahWali** [comment]
Yep that’s my post in the forum, wasn’t sure where i should post it

**Comment by terribleben** [comment]
Hey there, we upgraded react-native-maps to 0.21.0 in our latest SDK, 27.0.0. Can you check to see whether the problem still occurs? If so, it might be an upstream issue with the lib.

**Comment by AbdullahWali** [comment]
It's still not working, what do you mean by an upstream issue?

**Comment by brentvatne** [comment]
@AbdullahWali - an issue on https://github.com/react-community/react-native-maps which is what MapView in Expo is

**Comment by terribleben** [comment]
Our latest SDK (27.0.0, released yesterday) contains version 0.21.0 of [this maps lib](https://github.com/react-community/react-native-maps). By "upstream" I mean maybe it's a bug with that library. If you upgraded your project to Expo SDK 27 and still see the issue, then it's worth a look there.

**Comment by AbdullahWali** [comment]
@terribleben @brentvatne thank you for your input. Should I close this issue or should I keep it open until it is fixed?

Meanwhile for anybody having the same issue, you can use this workaround: 
instead of using initialRegion, use [onMapReady](https://github.com/react-community/react-native-maps/blob/master/docs/mapview.md#events) event to animate to the location you want 

                   onMapReady={()=>this.mapRef.animateToRegion({
                       latitude: location.coords.latitude,
                       longitude: location.coords.longitude,
                       latitudeDelta: 0.005,
                       longitudeDelta: 0.005
                   })}

**Comment by nthgol** [comment]
@terribleben  @AbdullahWali  im not seeing this resolved, just upgraded to 27

**Comment by alexcittadini** [comment]
How do I apply the iOS minor release. I am still seeing this issue
